controller/golden_files_test.go

```diff
@@ -0,0 +1,121 @@
+package controller_test
```

Comment:
Because not all structs can be compared with the JSON method. Any struct with private members cannot be compared this way. That's why I wanted to have it here.